compiler/ir/backend.common/src/org/jetbrains/kotlin/backend/common/serialization/IrModuleDeserializer.kt

```diff
@@ -44,7 +43,7 @@ abstract class IrModuleDeserializer(
     val logger: LoggingContext,
     val builtIns: IrBuiltIns,
     val symbolTable: SymbolTable
-) : IrDeserializer {
+) {
```

Comment:
That weird that `IrModuleDeserializer` is no more `IrDeserializer` when `KotlinIrLinker` is

Reply:
may be I need to rename `IrDeserializer` interface to `IrLinker` already.

Reply:
Whatever you like :)